graalpy/graalpy-custom-venv-guide/README.md

```diff
@@ -1,7 +1,7 @@
 # Using Custom GraalPy Virtual Environment in a Java Application
 
-While GraalPy provides a [python-embedding](https://central.sonatype.com/artifact/org.graalvm.python/python-embedding) package and GraalPy [Maven plugin](https://www.graalvm.org/latest/reference-manual/python/Embedding-Build-Tools/) simplify the required setup to ship Python packages
-as Java resources or in separate folders, sometimes users may want to install the Python packages manually
+While GraalPy provides a [python-embedding](https://central.sonatype.com/artifact/org.graalvm.python/python-embedding) package and GraalPy [Maven plugin](https://www.graalvm.org/latest/reference-manual/python/Embedding-Build-Tools/) to simplify the required setup to ship Python packages
```

Comment:
While GraalPy provides a [python-embedding](https://central.sonatype.com/artifact/org.graalvm.python/python-embedding) package and GraalPy [Maven plugin](https://www.graalvm.org/latest/reference-manual/python/Embedding-Build-Tools/) to simplify the required setup to ship Python packages as Java resources or in separate folders, sometimes developers may want to install the Python packages manually
using the GraalPy standalone distribution, and then use these manually installed packages in Java.